src/styles/tokens.css

```
@@ -1,414 +1,266 @@
+   * TYPOGRAPHY — Two families, one rule
+   *
+   *   --ff-display = Oswald (cinematic caps). Hero headlines + chip labels.
+   *                  Always uppercase, always tracked.
+   *   --ff-mono    = Space Mono (the HOTMESS look). Everything else: body,
+   *                  buttons, inputs, lists, nav.
+   *
+   * RULE: do not introduce a third family. If a surface "needs" something
+   * else, the design system is wrong — fix it here, not the surface.
+   * ════════════════════════════════════════════════════════════════════ */
+  --ff-display: 'Oswald', 'Bebas Neue', system-ui, -apple-system, sans-serif;
+  --ff-mono:    'Space Mono', 'SF Mono', Menlo, Monaco, Consolas, monospace;
```

Stylelint (17.11.1)

[error] Expected "Menlo" to be "menlo" (value-keyword-case)

(value-keyword-case)